test/test_fx.py

```diff
@@ -1161,5 +1161,13 @@ def forward(self, x):
 
         traced = symbolic_trace(NamedTupReturn())
 
+    def test_len(self):
+        class LenTest(torch.nn.Module):
+            def forward(self, x):
+                return torch.fx.len(x)
+
+        traced = symbolic_trace(LenTest())
+
+
```

Comment:
nit: extra blank line

And also a practical thing: adding tests to the very end can end up with merge conflicts if someone else adds a test similarly and tries to land it at the same time. I typically add tests somewhere in the middle of the file. (dumb solution, I know, but it works :) )

Reply:
Very useful, thanks James :)